src/Notepads/Notepads.csproj

```diff
@@ -449,6 +449,15 @@
   <ItemGroup>
     <None Include="Package.targets" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
You can remove these lines since it will be covered by <PRIResource Include="Strings\**\*.resw" />

Reply:
Only L449 to L451?

Reply:
@HasanAlyazidi only remove the lines you have added L452 to L460.

Reply:
@soumyamahunt @JasonStein done.